src/ts/utils/ShadowDomManager.ts

```diff
@@ -0,0 +1,124 @@
+import { DOM } from '../DOM';
+import { ShadowDomConfig } from '../UIConfig';
+import { prefixCss } from '../components/DummyComponent';
+
+/**
+ * Encapsulates Shadow DOM initialization.
+ */
+export class ShadowDomManager {
```

Comment:
Is there any reason to export (or to not export) the new `ShadowDomManager`  from `main.ts`, so that it can be used/accessed like other components?

Reply:
I guess that could be use-full: [455e554](https://github.com/bitmovin/bitmovin-player-ui/pull/808/commits/455e554488b8f1cb3556e5f7be99ce95c6bc3723)